registry-schema.adoc

```diff
@@ -0,0 +1,418 @@
+= Registry Schemas
+
+== extraDevfileEntries.yaml
+A file called extraDevfileEntries.yaml can be added under the root of the devfile registry source repository to add samples and stacks from other repositories to the registry. This file contains the location information on where the extra samples and stacks can be found.
+
+=== extraDevfileEntries.yaml schema
+- samples: schema[] - The sample list
+- stacks: schema[] - The stack list
+- name: string - The stack name
```

Comment:
Wondering if we need to make it clear that the items for `name` and below are for the `schema` type and not top level attributes. What do you think?

Reply:
Let me try to modify it to make it more clear.